tests/core/test_taipy.py

```diff
@@ -435,6 +435,28 @@ def test_get_scenarios(self, cycle):
             tp.get_scenarios(tag="tag")
             mck.assert_called_once_with("tag")
 
+    def test_get_scenarios_sorted(self):
+        scenario_1_cfg = Config.configure_scenario(name="B_scenario", id="scenario_3", tags=["banana", "kiwi"])
+        scenario_2_cfg = Config.configure_scenario(name="C_scenario", id="scenario_1", tags=["apple", "banana"])
+        scenario_3_cfg = Config.configure_scenario(name="A_scenario", id="scenario_2", tags=["apple", "kiwi"])
```

Comment:
Usually, the name and the tags are specific to a scenario instance. They are not passed when building a scenario configuration, but they are provided as parameters during the scenario creation.

You can have an example in my previous comment.

Reply:
All right, I see. Thank you for clarifying this. I will change it.